lib/datadog/tracing/contrib/sinatra/integration.rb

```diff
@@ -31,6 +31,14 @@ def new_configuration
             Configuration::Settings.new
           end
 
+          def subconfigure(tracing, sinatra)
+            tracing.instrument(
+              :rack,
+              service_name: sinatra.service_name,
+              distributed_tracing: sinatra.distributed_tracing,
+            )
```

Comment:
Most probably:
- other integrations are affected by this: Rails, Grape, etc...
- a similar pattern should be used by e.g `AppSec` integrations.